Doc/library/typing.rst

```diff
@@ -2330,7 +2330,7 @@ types.
       class XZ(X, Z): pass  # raises TypeError
 
       T = TypeVar('T')
-      class XT(X, Generic[T]): pass  # raises TypeError
+      class XT(X, Generic[T]): pass  # OK (Non-TypeDict class but Generic)
```

Comment:
Thanks for the PR! Since the next paragraph starts with "A ``TypedDict`` can be generic", and goes on to give several examples of generic TypedDicts, however, I think a better solution would be to just delete this example altogether

Reply:
> Thanks for the PR! However, since the next paragraph starts with "A `TypedDict` can be generic", and goes on to give several examples of generic TypedDicts, I think a better solution would be to just delete this example altogether

Hello @AlexWaygood, thanks for considering my PR, I am a bit confused, Should I delete all the examples and lines which mentions '**generic** TypedDict' because either way it's not compatible with python 3.12 onwards.
What I mean is, should I delete all these lines which are crossed in the following picture?
 
![changes](https://github.com/python/cpython/assets/102187286/e69be80f-8411-4011-8acc-646df4b4a8aa)

Reply:
> What I mean is, should I delete all these lines which are crossed in the following picture?

No, most of those lines are accurate documentation for Python 3.13, which is what the CPython `main` branch is :)

I think the only change required for this issue is to do this:

```diff
--- a/Doc/library/typing.rst
+++ b/Doc/library/typing.rst
@@ -2329,9 +2329,6 @@ types.

       class XZ(X, Z): pass  # raises TypeError

-      T = TypeVar('T')
-      class XT(X, Generic[T]): pass  # raises TypeError
-
    A ``TypedDict`` can be generic::
```

It looks like you might have closed this PR by accident. Would you like to open a new one? :)

Reply:
Yes, I am opening a new one with the changes you are asking for.